main.js

```diff
@@ -42,7 +42,14 @@ function parseSong(songString, settings) {
 		var songNote = songData[i].replace(".", "");
 		var match = pattern.exec(songNote);     
 		var dot = 0;
-
+		
+		//octave manager
+		if(isNaN(oct) == false){
```

Comment:
alright added oct to it
and disabled the refinement of octave, sorry I'm really tired